examples/utils.py

```diff
@@ -1,8 +1,61 @@
 # This module defines a set of common functions which are used in other examples.
 import time
 import datetime
-from tabulate import tabulate
+import os
+import importlib.util
+
+
+# IMPORTS
+def ensure_installed(*names):
```

Comment:
Nitpicks, but:

(1) more explanatory names are generally better - `ensure_package_is_installed`
(2) we could have `ensure_packages_are_installed` accepting an array of strings. If we pass nothing, it would just make sure that requirements are satisfied, if we pass strings - it will install them.